ami_environment.yml

```diff
@@ -0,0 +1,23 @@
+name: root
+channels:
+  - pytorch
+  - conda-forge
+dependencies:
+  - scipy
+  - numpy
+  - numba
+  - pandas
+  - jupyter
+  - matplotlib
+  - scikit-learn>=0.19
+  - mahotas
+  - scikit-image
+  - keras>=2.0.0
+  - h5py
+  - tensorflow
+  - pytorch-cpu
+  - torchvision-cpu
```

Comment:
Those pytorch-cpu and torchvision-cpu was copied from the ami_environment.yml file of the autism challenge. But I suppose here we want the gpu version? (I don't know what the default pytorch installs with conda?)